sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/TransportClient.cs

```diff
@@ -35,6 +35,11 @@ internal abstract class TransportClient : IAsyncDisposable
         ///
         public virtual Uri ServiceEndpoint { get; }
 
+        /// <summary>
+        /// The metrics related to the client.
+        /// </summary>
+        public virtual ServiceBusTransportMetrics Metrics { get; }
```

Comment:
Should we consider calling this `TransportMetrics`, since the associated type is strongly bound to that context?